builds/azure-pipelines/template-steps-build-test.yml

```diff
@@ -152,27 +151,58 @@ steps:
     checkStyleRunAnalysis: true
 
 - task: DotNetCoreCLI@2
-  displayName: '.NET Test'
+  displayName: '.NET Test on Mac (unit tests only)'
   env:
     TEST_SERVER: '$(testServer)'
     NODE_MODULES_PATH: '$(nodeModulesPath)'
     AZUREFUNCTIONS_SQLBINDINGS_TELEMETRY_OPTOUT: '1'
   inputs:
     command: test
     projects: '${{ parameters.solution }}'
-    arguments: '--configuration ${{ parameters.configuration }} ${{ parameters.testFilter }} --collect "Code Coverage" -s $(Build.SourcesDirectory)/test/coverage.runsettings'
-  condition: and(succeeded(), ne(variables['Agent.OS'], 'linux'))
+    # Disabling integration tests on macOS due to issues with running MSSQL on Docker
+    # We need to set up a self-hosted agent with Docker running by default: https://github.com/microsoft/azure-pipelines-tasks/issues/12823
+    arguments: '--configuration ${{ parameters.configuration }} --filter FullyQualifiedName!~Integration'
+  condition: and(succeeded(), eq(variables['Agent.OS'], 'Darwin'))
+
+- task: DotNetCoreCLI@2
+  displayName: '.NET Test on Windows (unit tests and CSharp integration tests)'
+  env:
+    TEST_SERVER: '$(testServer)'
+    NODE_MODULES_PATH: '$(nodeModulesPath)'
+    AZUREFUNCTIONS_SQLBINDINGS_TELEMETRY_OPTOUT: '1'
+    # Only running CSharp integration tests to get accurate code coverage numbers
+    LANGUAGES_TO_TEST: 'CSharp'
+  inputs:
+    command: test
+    projects: '${{ parameters.solution }}'
+    # Skip any non-CSharp integration tests
+    arguments: '--configuration ${{ parameters.configuration }} --filter FullyQualifiedName!~NoPropertiesThrows --collect "Code Coverage" -s $(Build.SourcesDirectory)/test/coverage.runsettings'
```

Comment:
The only supported language for the NoPropertiesThrows test is JavaScript so this error was being thrown if I didn't skip this test for the CSharp run:
`   System.InvalidOperationException : No data found for Microsoft.Azure.WebJobs.Extensions.Sql.Tests.Integration.SqlOutputBindingIntegrationTests.NoPropertiesThrows`

Reply:
Add this as a comment